remoteappmanager/static/js/vue/filters.js

```diff
@@ -1,20 +1,20 @@
 define([
     "components/vue/dist/vue",
     "urlutils"
-], function (Vue, urlutils) {
+], function (Vue, urlUtils) {
```

Comment:
google style guide says lowercase for filenames.. so no, the file is urlutils...

Leave it as is.